### PR TITLE
[Build][Bugfix] Fix build failed when GC_ENABLE_DNNL=ON

### PR DESCRIPTION
**Pull request by huanghaixin008**
Tracking [#221](https://github.com/intel/graph-compiler/issues/221)

### CODE REVIEW

File: cmake/onednn.cmake [hunk 1]
```diff
@@ -18,9 +18,9 @@ if (NOT DEFINED GC_DNNL_INCLUDES)
     # This allows to generate headers from *.in without adding the library to the build.
     # If the build is required, remove this and the SKIP_ADD option above.
     if (DEFINED CMAKE_GENERATOR)
-        set(GENERATOR_FLAG "-G ${CMAKE_GENERATOR}")
+        set(GENERATOR_FLAG "${CMAKE_GENERATOR}")
     endif ()
-    execute_process(COMMAND ${CMAKE_COMMAND} ${GENERATOR_FLAG}
+    execute_process(COMMAND ${CMAKE_COMMAND} -G ${GENERATOR_FLAG}
```
Comment (dchigarev) [inline cmake/onednn.cmake:23]: what if `CMAKE_GENERATOR` is not defined? It seems that we'll get here an invalid command anyway: 
```
cmake -G _empty_ -Wno-dev ...
```

Comment (huanghaixin008) [reply cmake/onednn.cmake]: Fixed by adding an extra var for `-G`.

**Review by dchigarev — APPROVED** [review]
@huanghaixin008 
Have you tested that it fixes the issue for you? Because for me building with `-DGC_ENABLE_DNNL=ON` works with and without your fix

**Review by kurapov-peter — APPROVED** [review]

**Comment by huanghaixin008** [comment]
> @huanghaixin008 Have you tested that it fixes the issue for you? Because for me building with `-DGC_ENABLE_DNNL=ON` works with and without your fix

Yes I did. Two other teammates and me cannot build on latest `main` branch, and with this fix I can build.

**Pull request merged**